src/AeroSharp/DataAccess/Policies/MapConfigurationToMapPolicyMapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Aerospike.Client;
+using System.Linq;
+
+namespace AeroSharp.DataAccess.Policies;
+
+/// <summary>
+///     Maps a MapConfiguration to a MapPolicy to be used with Map Operations
+/// </summary>
+internal static class MapConfigurationToMapPolicyMapper
+{
+    public static MapPolicy MapToPolicy(MapConfiguration mapConfiguration)
+    {
+        var flagsArray = new[]
+        {
+            mapConfiguration.CreateOnly ? MapWriteFlags.CREATE_ONLY : 0,
+            mapConfiguration.UpdateOnly ? MapWriteFlags.UPDATE_ONLY : 0,
```

Comment:
Are these values mutually exclusive?

Reply:
If so, we may want to add a rule to the validator

Reply:
The comments here seem to indicate that they would be mutually exclusive: https://github.com/aerospike/aerospike-client-csharp/blob/80f15b7ffa44af30a8f4e1eeb0a28a1e7cada9a5/AerospikeClient/CDT/MapWriteFlags.cs

I'll see about adding our own validation against that. 😄

Reply:
Created a validator and doing validation within our builder to catch issues early. LMK what you think!